lisa/sut_orchestrator/azure/features.py

```diff
@@ -446,7 +446,7 @@ def reload_module(self) -> None:
         modprobe_tool = self._node.tools[Modprobe]
         modprobe_tool.reload(["hv_netvsc"])
 
-    @retry(tries=60, delay=2)
+    @retry(tries=60, delay=10)
```

Comment:
60 times * 10 seconds. It means 10 minutes. Is it too long?

Reply:
I would rather have a stable test with a slow worst-case than a uniformly fast test that is unstable.

Reply:
But yes I think 10 is a safe maximum.